tests/unit/s2n_override_openssl_random_test.c

```diff
@@ -144,7 +144,7 @@ int main(int argc, char **argv)
     END_TEST();
 }
 
-#else /* defined(OPENSSL_IS_BORINGSSL) */
+#else /* defined(OPENSSL_IS_BORINGSSL) || defined(OPENSSL_IS_AWSLC) */
```

Comment:
I would just remove this comment. It's going to be hard to keep up to date with S2N_LIBCRYPTO_SUPPORTS_CUSTOM_RAND, and I think it's already inaccurate.

Reply:
Removed